map/page.js

```diff
@@ -176,13 +178,23 @@ function updateMap(data) {
   } catch (err) {
     console.warn('cannot fit bounds');
   }
-  amap = map;
-  const rowId = selectedRowId;
-  if (rowId && popups[rowId]) {
-    var marker = popups[rowId];
-    if (!marker._icon) { marker.__parent.spiderfy(); }
-    marker.openPopup();
+  function makeSureSelectedMarkerIsShown() {
+    const rowId = selectedRowId;
+    if (rowId && popups[rowId]) {
+      var marker = popups[rowId];
+      if (!marker._icon) { marker.__parent.spiderfy(); }
+      marker.openPopup();
+    }
   }
+  map.on('zoomend', () => {
+    // Should reshow marker if it has been lost, but I didn't find a good
+    // event to trigger that exactly. A small timeout seems to work :-(
+    // TODO: find a better way; also, if user has changed selection within
+    // the map we should respect that.
+    setTimeout(makeSureSelectedMarkerIsShown, 500);
```

Comment:
This makes the map move always to show the marker, I found that this fixes that issue.

`171: marker.bindPopup(title, {autoPan: false});`

Reply:
Hmm, Natalie seemed to specifically like that "feature" - `I like that if you zoom in and it's no longer in the window, it adjusts your view of the map to show you the selected location` - do you see a problem with it?

Reply:
Ah ok, I thought that it would be seen as a bug - because you can't zoom anywhere, but maybe it is ok, this map's purpose is to show those points after all.